### PR TITLE
hw3-done, hw4-fixed

### PR DESCRIPTION
**Pull request by cyberdayag**
ready for review

### CODE REVIEW

**Comment by Alex-Shev75** [comment]
Alexey,  check pls

**Comment by cyberdayag** [comment]
Alexey, i created one repository for all homework and added here 3 and 4 works in one branch. Check pls.

File: hw4/hw-lesson-4.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""
+Homework-4: text type, even/odd, length
+"""
+
+text = input('Please enter a word or a number - ')
+a = text
+if text.isdigit():
+    text = int(text)
+    x = text % 2
```
Comment (klymenok) [inline hw4/hw-lesson-4.py]: There is no big need to create a separate variable here. You can put `text % 2` right into `if` condition

**Review by klymenok — CHANGES_REQUESTED** [review]
Good, but pay attention to the comment

**Comment by cyberdayag** [comment]
fixed

**Comment by Alex-Shev75** [comment]
hw4 fixed, hw5 added for review

**Pull request merged**